src/modules/previewpane/common/controls/WebBrowserExt.cs

```diff
@@ -54,6 +54,7 @@ public object InvokeMember(string name, BindingFlags invokeAttr, Binder binder,
 
                 if (name != null && name.Equals(DISPIDAMBIENTDLCONTROL, StringComparison.CurrentCulture))
                 {
+                    // Using CurrentCulture since this is user facing
```

Comment:
@somil55 do you know if this is actually user facing? It seems to be converting strings to some flags, so it feels like an Ordinal/InvariantCulture case to me.

Reply:
I don't think this is user-facing as this seems to be used for web browser configurations. We can update it to use ordinal culture.

Reply:
Changing to Invariant since CultureInfo does not have Ordinal. Thanks!